packages/fluent-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -97,7 +97,7 @@ const SelectWidget = ({
   const uiProps = _pick(options.props || {}, allowedProps);
   return (
     <>
-      <Label>{label || schema.title}</Label>
+      {typeof uiProps.onRenderLabel === "undefined" && <Label>{label || schema.title}</Label>}
```

Comment:
Thanks, can you add a test?

Reply:
@epicfaace Would you be able to point me in the right directions to do a test, or maybe just make a small one in the PR to show me how and i can be better at in the future.

I am a big user of FluentUI and using RJSF for a larger project atm, so i expect that i can do some contributions.

Reply:
Sure, see this folder for some existing tests: https://github.com/rjsf-team/react-jsonschema-form/tree/master/packages/fluent-ui/test